src/protocol/boards/aiy/api.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Please add a check for the used features or use an pseudo device like its used for the other hardware integration.